core/src/backend/ui.rs

```diff
@@ -81,7 +81,7 @@ pub trait UiBackend: Downcast {
     /// Displays a message about an error during root movie download.
     /// In particular, on web this can be a CORS error, which we can sidestep
     /// by providing a direct .swf link instead.
-    fn display_root_movie_download_failed_message(&self);
+    fn display_root_movie_download_failed_message(&self, _invalid_swf: bool);
```

Comment:
Oh noo, the "boolean trap"! :sweat_smile: 

https://ariya.io/2011/08/hall-of-api-shame-boolean-trap

> it’s almost invariably a mistake to add a bool parameter to an existing function

Okay, don't take this that seriously, I'm also prone to falling victim to it - usually purposefully.
It's probably fine here.

Reply:
I think the least bad alternative would be an enum-ish parameter. Maybe named integer constants, or string literals. But given how any reasonable IDE nowadays will tell you the name of the parameter as which you are passing that `false` immediately on hover anyway, there's not much point.